templates/repo/view_file.tmpl

```diff
@@ -110,10 +110,10 @@
 				</table>
 				<div class="code-line-menu ui vertical pointing menu tippy-target">
 					{{if $.Permission.CanRead $.UnitTypeIssues}}
-						<a class="item ref-in-new-issue" href="{{.RepoLink}}/issues/new?body={{.Repository.HTMLURL}}{{printf "/src/commit/"}}{{PathEscape .CommitID}}/{{PathEscapeSegments .TreePath}}" rel="nofollow noindex">{{.locale.Tr "repo.issues.context.reference_issue"}}</a>
+						<a class="item ref-in-new-issue" href="{{.RepoLink}}/issues/new?body={{.Repository.Link}}{{printf "/src/commit/"}}{{PathEscape .CommitID}}/{{PathEscapeSegments .TreePath}}" rel="nofollow noindex">{{.locale.Tr "repo.issues.context.reference_issue"}}</a>
```

Comment:
Here seems incorrect.

When you are referring a file link in an issue, you need the full URL at the moment.

IIRC

`https://try.gitea.io/wxiaoguang/test/src/commit/dc757ee96cf02d80dce4afd41e957aa2a2e341f3/b.go#L1` works

But `/wxiaoguang/test/src/commit/dc757ee96cf02d80dce4afd41e957aa2a2e341f3/b.go#L1` doesn't work in the issue body.

Reply:
I will revert this change and add a comment here.

Reply:
Done.